.changeset/witty-seas-meow.md

```diff
@@ -0,0 +1,9 @@
+---
+'@aws-amplify/ui-vue': major
+---
```

Comment:
I created a different changeset for each framework because the texts we are deprecating is slightly different for each of them.